cluster/gce/util.sh

```diff
@@ -1713,7 +1713,7 @@ function create-subnetworks() {
       --secondary-range "services-default=${SERVICE_CLUSTER_IP_RANGE}"
     echo "Created subnetwork ${IP_ALIAS_SUBNETWORK}"
   else
-    if ! echo ${subnet} | grep --quiet secondaryIpRanges ${subnet}; then
+    if ! echo ${subnet} | grep -qs secondaryIpRanges ${subnet} 2>/dev/null; then
```

Comment:
Looks like the problem is with the second `${subnet}`:

`if ! echo ${subnet} | grep -qs secondaryIpRanges; then` should work

Reply:
You are right! Fixed it. PTAL